docker-compose.yml

```diff
@@ -41,8 +41,8 @@ services:
     image: nginx:1.13
     restart: unless-stopped
     ports:
-      - "80:80"
-      - "443:443"
+      - "${OPENEDX_NGINX_PORT:-80}:80"
+      - "${OPENEDX_NGINX_TLSPORT:-443}443:443"
```

Comment:
Can I suggest naming the variables `NGINX_HTTP_PORT` and  `NGINX_HTTPS_PORT`? We don't need the `OPENEDX` prefix, as we are inside the openedx-docker repo, and HTTPS is more understandable (IMHO) than TLS.

Also, there is a typo in the 443 declaration. It should be:

    - "${OPENEDX_NGINX_TLSPORT:-443}:443"

Reply:
Done